client/ionic/android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@color/ic_launcher_background"/>
-    <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
+    <foreground android:drawable="@mipmap/ic_launcher"/>
```

Comment:
forgive my android ignorance here, but what's the reason for the change here and `ic_launcher_round.xml`?

Reply:
I'm not 100% sure, but my understanding is that Android offers the option to ship both a square oldschool icon and the (optional) modern/rounded icon which is actually two layers, the forground can be masked and hovers over a background (in a way to create a cool effect) but in our case we have just the normal icon, without any foreground/background stuff so I just called it ic_launcher, which I _think_ is the convention. But if someone else knows this Android stuff better please do add some thoughts here.

Reply:
Oh, the short answer to your question is: if I didn't rename this the build fails.